lib/cocoapods-core/specification.rb

```diff
@@ -514,6 +520,23 @@ def convert_keys_to_string(value)
       result
     end
 
+    # Converts the keys of the given hash to a string.
+    #
+    # @param  [Object] value
+    #         the value that needs to be stripped from the Symbols.
+    #
+    # @return [Hash] the hash with the strings instead of the keys.
+    #
+    def convert_keys_to_symbol(value)
```

Comment:
method moved from `root_attribute_accessors.rb` into `specification.rb` because for the first time we have a non root attribute that uses symbols. No change in the method itself. There were no tests for it either.